multiapps-controller-web/src/main/java/org/cloudfoundry/multiapps/controller/web/api/impl/FilesApiServiceImpl.java

```diff
@@ -136,6 +136,7 @@ public ResponseEntity<FileMetadata> uploadFile(MultipartHttpServletRequest reque
 
     @Override
     public ResponseEntity<Void> startUploadFromUrl(String spaceGuid, String namespace, FileUrl fileUrl) {
+        System.setProperty(Constants.RETRY_LIMIT_PROPERTY, "0");
```

Comment:
Is it necessary to set this each time, or can it be moved to a static block instead?